extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -147,6 +157,32 @@ export class BaseSlides extends BaseCarousel {
         this.autoplayDelay_);
   }
 
+  /**
+   * Called on user interaction to toggle the autoplay feature.
+   */
+  toggleAutoplay(toggleOn) {
```

Comment:
make private `toggleAutoplay_` and add JS doc for the @param

Reply:
Done